lib/index.js

```diff
@@ -21,7 +21,7 @@ function inspect(basePath, fileName, options) {
     }
     // we want to load the json file as well; usually composer.json
     composerJsonObj = loadJsonFile(basePath,
-      fileName.split('.').shift() + '.json');
+      fileName.split('.').slice(0, -1).join('.') + '.json');
```

Comment:
Wow we were quite naive here, thinking there's only one `.` in the path. Can I ask you to amend this so that the `path` module is used, something like
```
composerJsonObj = loadJsonFile(path.resolve(basePath, path.dirname(fileName), 'composer.json'));
```